build-tools/build-image/cross/VERSION

```diff
@@ -1 +1 @@
-v1.7.1-0
+v1.7.1-2
```

Comment:
just FYI @jessfraz, this will conflict with https://github.com/kubernetes/kubernetes/pull/35716